docs/reference/dependencies.md

```diff
@@ -18,6 +18,7 @@ import TabItem from "@theme/TabItem";
 {label: 'Identity', value: 'identity', },
 {label: 'Optimize', value: 'optimize', },
 {label: 'Desktop Modeler', value: 'modeler', },
+{label: 'Web Modeler (Beta)', value: 'web-modeler', },
```

Comment:
I assume we should not say "Beta" here, since this may be misleading that also on SaaS, it's only a beta.